overlays/python38/Dockerfile

```diff
@@ -11,6 +11,26 @@ ENV \
     # Path to be used in other layers to place s2i scripts into
     STI_SCRIPTS_PATH=/opt/app-root/builder
 
+USER root
```

Comment:
can you please copy the changes and create an new subdir:
`overlays/python38-internal`
and move this changes there

Reply:
@harshad16 moved!